mitodl/ocw-studio
# chore(e2e): update fixtures and fix typo

### What are the relevant tickets?
Part of https://github.com/mitodl/ocw-hugo-themes/pull/1364

### Description (What does it do?)
This PR:
- adds test fixtures for https://github.com/mitodl/ocw-hugo-themes/pull/1364
- fixes a typo in `export_test_sites` command 


### How can this be tested?

1. Navigate to your local setup of ocw-studio.
2. Checkout `hussaintaj/themes-pr-1364`.
3. Start Studio.
4. Run
    ```sh
    docker compose exec web ./manage.py loaddata test_site_fixtures/test_website_content.json
    ```
5. Expect the above command to succeed.
6. Expect the following page to exist [http://localhost:8043/sites/ht-test-2024/type/external-resource/edit/0c79321c-611c-4c6c-a7cf-9605704373c9/](http://localhost:8043/sites/ht-test-2024/type/external-resource/edit/0c79321c-611c-4c6c-a7cf-9605704373c9/)
7. Run
   ```
   docker compose exec web ./manage.py export_test_sites
   ```
8. Expect the above command to succeed.
